### ISSUE TITLE
Doesn't work if non-circular object is 'in front of' the circular reference

### ISSUE THREAD

**Issue by nathanfdunn** [tjmehta/is-circular#2]
``` javascript
var isCircular = require('is-circular');

var obj1 = {};
obj1.self = obj1;
obj1.other = {};
console.log( isCircular(obj1) );    //true

var obj2 = {};
obj2.other = {};
obj2.self = obj2;
console.log( isCircular(obj2) );    //false
```

**Comment by tjmehta** [comment]
Oh wow.

**Comment by tjmehta** [comment]
Fixed v1.0.1 Thanks!
